src/search/searchbar.js

```diff
@@ -5,7 +5,9 @@ export default class searchbar extends Component {
         return (
             <div>
                 <label>Search By Name:</label>
-                <input type='text' onChange={this.props.handleChange} />
+                <input type='text'
+                    value={this.props.currentValue}
+                    onChange={this.props.handleChange} />
                 <button onClick={this.props.buttonClick}>start</button>
```

Comment:
I noticed the Start button wasn't quite working. It's called buttonClick here. If you change it to this.props.handleClick the button should work! :)